src/Symfony/Bundle/TwigBundle/ErrorRenderer/TwigHtmlErrorRenderer.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\TwigBundle\ErrorRenderer;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\ErrorRenderer\ErrorRenderer\HtmlErrorRenderer;
+use Symfony\Component\ErrorRenderer\Exception\FlattenException;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Twig\Environment;
+use Twig\Error\LoaderError;
+use Twig\Loader\ExistsLoaderInterface;
+
+/**
+ * @author Yonel Ceruto <yonelceruto@gmail.com>
+ */
+class TwigHtmlErrorRenderer extends HtmlErrorRenderer
```

Comment:
missing phpdoc what this class does compared to the extended one. it's not obvious

Reply:
I would also suggest to implement this with composition instead of inheritance.

Reply:
please use composition (inject HtmlErrorRenderer into TwigHtmlErrorRenderer) and add in the phpdoc that it only uses the custom Twig templates in non-debug mode and otherwise falls back to HtmlErrorRenderer